### PR TITLE
add session

### PR DESCRIPTION
**Pull request by moklidia**

### CODE REVIEW

File: app/controllers/web/sessions_controller.rb [hunk 1]
```diff
@@ -1,4 +1,26 @@
 class Web::SessionsController < Web::ApplicationController
   def new
+  	@session = Session.new
+  end
+
+  def create
+  	@session = Session.new(session_params)
```
Comment (artempartos) [inline app/controllers/web/sessions_controller.rb]: indent

File: test/controllers/web/sessions_controller_test.rb [hunk 2]
```diff
@@ -6,4 +6,19 @@ class Web::SessionsControllerTest < ActionDispatch::IntegrationTest
     assert_response :success
   end
 
+  test "should post create" do
+  	password = generate(:password)
```
Comment (artempartos) [inline test/controllers/web/sessions_controller_test.rb]: please, run rubocop before submit new PR

Comment (moklidia) [reply test/controllers/web/sessions_controller_test.rb]: Done here: https://github.com/moklidia/task-manager/pull/10/

**Comment by artempartos** [comment]
👍

**Pull request closed without merging**